release.nix

```diff
@@ -6,11 +6,17 @@
 , checkMaterialization ? false
 }:
 let
+  traceNames = prefix: builtins.mapAttrs (n: v:
```

Comment:
probably don't want to commit this? not sure

Reply:
I have found it very useful when debugging hydra eval errors.  Some times you don't know what it was evaluating when the error happened (since the hydra log does not include the `stdout` output of `hydra-eval-jobs`).  With this you can tell what you need to try with `nix-build release.nix -A` or what to look for in `nix repl release.nix` to reproduce the error.

Reply:
Fine with me!